charts/jenkinsio/templates/nginx-configmap.yaml

```diff
@@ -30,6 +30,8 @@ data:
             rewrite ^/$ https://www.jenkins.io/$lang$1;
           }   
       }
+      
+      add_header X-Frame-Options "DENY";
```

Comment:
do you need to enable it on the chinese version too?

https://github.com/jvz/charts/blob/patch-1/charts/jenkinsio/templates/zh-configmap.yaml

Reply:
Perhaps once Chinese script kiddies start to pester us about that subdomain, too :P